src/main/kotlin/exercise2/task5/CreateUserDSL.kt

```diff
@@ -49,11 +49,14 @@ internal data class Address(
  */
 
 internal fun user(initUser: User.() -> Unit): User {
-    TODO("Implement me!!!")
+    val korisnik = User().apply(initUser)
```

Comment:
That's the solution! It can also be a one-line expression since you can return `User().apply(initUser)` as a function result. 

In the future, please declare your variables in English.